src/daemon/rpmostreed-sysroot.cxx

```diff
@@ -549,11 +549,21 @@ rpmostreed_sysroot_init (RpmostreedSysroot *self)
   /* Only use polkit when running as root on system bus; self-tests don't need it */
   if (!self->on_session_bus)
     {
-      g_autoptr (GError) local_error = NULL;
-      self->authority = polkit_authority_get_sync (NULL, &local_error);
-      if (self->authority == NULL)
+      // See https://github.com/coreos/rpm-ostree/issues/4323 and
+      // https://github.com/coreos/fedora-coreos-tracker/issues/334 and
+      // https://lists.freedesktop.org/archives/dbus/2023-March/018252.html Basically, starting
+      // polkit can time out on slow systems; we want to just keep retrying so that we hit the
+      // systemd unit activation timeout (90s); crucially whatever it's configured to be, so that
+      // slow systems can bump that timeout.
+      while (self->authority == NULL)
         {
-          errx (EXIT_FAILURE, "Can't get polkit authority: %s", local_error->message);
+          g_autoptr (GError) local_error = NULL;
+          self->authority = polkit_authority_get_sync (NULL, &local_error);
+          if (self->authority == NULL)
```

Comment:
My only hesitation with this is that if `polkit_authority_get_sync` is failing not because of a timeout but something else (e.g. misconfiguration), we're now spamming the journal in a hot loop until systemd kills us.

Two independent suggestions:
1. only retry if the error we got is actually that we timed out; not sure if we have access to the error domain and code here to make this painless
2. add a minimal sleep between tries